src/styles/inherited-scss/components/filter-panel/_filter-panel.scss

```diff
@@ -18,11 +18,15 @@
     box-shadow: 0px 3px 8px #d4dadd;
   }
 
-  @media screen and (max-width: 768px) {
-    width: 320px;
+  @media screen and (max-width: 800px) {
```

Comment:
I had to increase the width tick to avoid horizontal scrolling because of width overflow here:

<img width="880" alt="Screenshot 2022-02-13 at 10 40 49" src="https://user-images.githubusercontent.com/608862/153749503-c4a5fdb3-4a90-41d2-99e8-b65daaca2b41.png">